src/aioice/ice.py

```diff
@@ -882,8 +885,14 @@ async def get_component_candidates(
             candidates.append(protocol.local_candidate)
         self._protocols += host_protocols
 
-        # query STUN server for server-reflexive candidates (IPv4 only)
+        return candidates, host_protocols
+
+    async def query_stun_server(
+        self, host_protocols: List[StunProtocol], timeout: int = 5
+    ) -> Optional[List[Candidate]]:
+        candidates = None
```

Comment:
Why not simply initialise candidates to an empty list instead of having to deal with None?